test/hosted-field.test.js

```diff
@@ -0,0 +1,42 @@
+import assert from 'assert';
+import bowser from 'bowser';
+
+import { fixture } from './support/fixtures';
+import { initRecurly } from './support/helpers';
+import { HostedField } from '../lib/recurly/hosted-field';
+import { Recurly } from '../lib/recurly';
+
+describe('HostedField', () => {
+  fixture('minimal');
```

Comment:
Instead of calling `fixture` here, let's use `applyFixtures` and set `this.ctx.fixture` in order to have it built and torn down per test.